pkg/events/events.go

```diff
@@ -5975,6 +5976,36 @@ var Definitions = eventDefinitions{
 					{MapName: "prog_array", MapIndexes: []uint32{tailKernelWrite}, ProgName: "trace_ret_kernel_write_tail"},
 					{MapName: "prog_array", MapIndexes: []uint32{tailSendBin}, ProgName: "send_bin"},
 				},
+				Events: []eventDependency{
+					{VfsWrite},
+					{VfsWritev},
+					{KernelWrite},
+				},
+			},
+		},
+		CaptureFileRead: {
+			ID32Bit:  sys32undefined,
+			Name:     "capture_file_read",
+			Internal: true,
+			Probes: []probeDependency{
+				{Handle: probes.VfsRead, Required: true},
+				{Handle: probes.VfsReadRet, Required: true},
+				{Handle: probes.VfsReadV, Required: false},
+				{Handle: probes.VfsReadVRet, Required: false},
+			},
+			Dependencies: dependencies{
+				TailCalls: []TailCall{
+					{MapName: "prog_array", MapIndexes: []uint32{tailVfsRead}, ProgName: "trace_ret_vfs_read_tail"},
+					{MapName: "prog_array", MapIndexes: []uint32{tailVfsReadv}, ProgName: "trace_ret_vfs_readv_tail"},
+					{MapName: "prog_array", MapIndexes: []uint32{tailSendBin}, ProgName: "send_bin"},
+				},
+				Events: []eventDependency{
+					{VfsRead},
+					{VfsReadv},
+				},
+				KSymbols: &[]kSymbolDependency{
+					{Symbol: "pipefifo_fops", Required: true},
```

Comment:
Where do we use that?

Reply:
We need it to know if a file is a pipe or not for the file type filter.
https://github.com/aquasecurity/tracee/pull/2356/files#diff-c046893996f2e2a6e44a8caac7a14abbf47b47486ee5ceaa9029c34fc5f94a99R393